### PR TITLE
AND-9 & AND-10

### PR DESCRIPTION
**Pull request by mzielu**

### CODE REVIEW

File: app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2014 Pedro Vicente Gómez Sánchez.
```
Comment (grzanka) [inline app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java]: Is it part of come library ? Looks like copy-pasted from the web.
If this is a part of some external lib/entity can't it be used without copy-pasting ?

Comment (mzielu) [reply app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java]: Yeah, I have written on Trello that I will use existing library with small corrections (adjusted to our code). What should I do in this situation?

Comment (grzanka) [reply app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java]: Add a comment to the code mentioning from where this library comes from (i.e. URL), add a comment to places which were modified (if reasonable) and explain in comments why was it modified.

Comment (mzielu) [reply app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java]: @grzanka Done. Please check if code is correct now.

**Comment by grzanka** [comment]
Now it looks OK.

**Pull request merged**